guides/WRITING-DOCUMENTATION.md

```diff
@@ -68,14 +68,15 @@ The description should describe what pattern does or should be used for in plain
-| **`@responsive`**           | `@responsive`                                                | Renders breakpoint toggles for the markup example.                                                                                      |
-| **`@status [NAME]`**        | `@status Draft`                                              | Displays a status badge. `[NAME]` must be one of the following values: `Draft`, `Work in progress`, `Ready`, `Deprecated`.              |
-| **`@uswds [URL]`**          | `@uswds https://designsystem.digital.gov/components/button/` | `[URL]` is the link to the corresponding US Web Design System component.                                                                |
+| flag name                   | example                                                     | flag description                                                                                                                        |
+| --------------------------- | ----------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| **`@hide-section`**         | `@hide-section`                                             | Hides an entire documentation section. If used on a page's top level section, it will hide the entire page                              |
+| **`@hide-markup`**          | `@hide-markup`                                              | Hides the code snippet.                                                                                                                 |
+| **`@react-props [NAME]`**   | `@react-props Button.jsx`                                   | Displays the React prop documentation for a component. `[NAME]` is the filename of the react component where the Proptypes are defined. |
+| **`@react-example [NAME]`** | `@react-example Button.example.jsx`                         | Displays the example file using this React component. `[NAME]` is the filename of the react example to be displayed.                    |
+| **`@responsive`**           | `@responsive`                                               | Renders breakpoint toggles for the markup example.                                                                                      |
+| **`@status [NAME]`**        | `@status Draft`                                             | Displays a status badge. `[NAME]` must be one of the following values: `Draft`, `Ready`, `Deprecated`.                                  |
+| **`@uswds [URL]`**          | `@uswds https://designsystem.digital.gov/components/alerts` | `[URL]` is the link to the corresponding US Web Design System component.                                                                |
```

Comment:
Under  Writing general documentation section, add this new `hide-section: true` optional property. Also noticed `hide-markup` property is missing from this section too.

Reply:
Yeah, so our KSS and Markdown flags are actually the same, and so these two tables arent technically correct. I would prefer not to update the Markdown flag table in this PR, as theres several flags that arent updated across both tables. In the future we will be removing KSS anyway, so we will eventually have a single table of frontmatter values